package.json

```diff
@@ -40,9 +40,9 @@
     "ts-loader": "^9.4.2",
     "typedoc": "^0.17.7",
     "typescript": "^4.9.4",
-    "webpack": "^5.75.0",
-    "webpack-cli": "^5.0.1",
-    "webpack-dev-server": "^4.11.1"
+    "webpack": "^5.95.0",
+    "webpack-cli": "^5.1.4",
+    "webpack-dev-server": "^5.1.0"
```

Comment:
@JAHMCR as this is a breaking change, did you verify devs can still do `npm run dev` and it works?

Reply:
Sorry, pushed the wrong package version, I'm working on that breaking change implementation